tool/tctl/common/auth_command.go

```diff
@@ -778,15 +778,15 @@ func (a *AuthCommand) generateUserKeys(ctx context.Context, clusterAPI auth.Clie
 	}
 	key.TrustedCA = auth.AuthoritiesToTrustedCerts(hostCAs)
 
-	networkConfig, err := clusterAPI.GetClusterNetworkingConfig(ctx)
-	if err != nil {
-		return trace.Wrap(err)
+	// Is TLS routing enabled?
+	proxyListenerMode := a.config.Auth.NetworkingConfig.GetProxyListenerMode()
+	if networkConfig, err := clusterAPI.GetClusterNetworkingConfig(ctx); err == nil {
+		proxyListenerMode = networkConfig.GetProxyListenerMode()
```

Comment:
Can we just output a debug log here so it's clear another mode of behaviour is in play ?